packages/web/src/pages/settings-page/components/desktop/SettingsPage.tsx

```diff
@@ -381,4 +389,10 @@ class SettingsPage extends Component<SettingsPageProps, SettingsPageState> {
   }
 }
 
-export default SettingsPage
+function mapDispatchToProps(dispatch: Dispatch) {
```

Comment:
this should go in SettingsPageProvider to follow the existing pattern of connected "provider" components

Reply:
also, this leaves out the mobile web version of this page right?

Reply:
ah yeah that makes sense, well mobile just using dispatch manually itself, ie they both handle it, but i can thread it through both to keep the pattern

Reply:
oh actually they are different since mobile signs out in the confirmation drawer...